enhancements/update/manifest-install-levels.md

```diff
@@ -0,0 +1,166 @@
+---
+title: manifest-install-levels
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
```

Comment:
since this is affecting control plane, I would expect this to be reviewed by @openshift/api-reviewers and the control plane team (@sttts @deads2k)

Reply:
> since this is affecting control plane, I would expect this to be reviewed by...

The more, the merrier.  But I'm not clear on the control-plane connection.  Can you clarify?